Correct instructions to help readers find template (#148)
madewithkoji/docs: src/asciidoc/tutorials/getting-started/your-first-remix.adoc

There are templates focused on social media engagement, memes, and games.
New templates are being added all the time by developers like you.

> Scroll down until you see 'Engage Your Audiences', click on 'See All', then scroll down until you find the Bouncy Blast template.

Reviewer: ```suggestion
Scroll down until you see *Engage Your Audiences*, click *See All*, then scroll down until you find the Bouncy Blast template.
```
